.github/workflows/full_kubeflow_integration_test.yaml

```diff
@@ -0,0 +1,505 @@
+name: Full Kubeflow Integration Test
+
+on:
+  workflow_dispatch:  
+  schedule:
+    - cron: '0 0 * * 0'  
+  pull_request:
+    paths:
+      - example/kustomization.yaml
+      - .github/workflows/full_kubeflow_integration_test.yaml
+      - apps/pipeline/upstream/**
+      - apps/katib/**
+      - apps/centraldashboard/**
+      - apps/jupyter/**
+      - apps/profiles/**
+      - apps/tensorboard/**
+      - apps/training-operator/**
+      - apps/kserve/**
+      - common/**
+      - tests/gh-actions/**
+  push:
+    branches:
+      - master
+      - v*-branch
+      - 'releases/**'
+
+jobs:
+  build:
+    name: Full E2E Integration Test
+    runs-on:
+      labels: ubuntu-latest-16-cores
+    env:
+      KIND_CLUSTER_NAME: kubeflow
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+        
+    - name: Install Python dependencies
+      run: |
+        pip install pytest kubernetes kfp==2.11.0 kserve pytest-timeout pyyaml requests
+
+    - name: Install kubectl and kustomize
```

Comment:
Please reference the existing scripts and files under / tests everywhere, do not just copy it over.

Reply:
The goal is not to copy the readme, but  to reuse and recombine our existing tests. Only what is missing can come from the readme. So the readme comes last, after you have combined the workflows here https://github.com/kubeflow/manifests/tree/master/.github/workflows using the scripts and objects from here https://github.com/kubeflow/manifests/tree/master/tests